include/fiction/algorithms/simulation/sidb/quicksim.hpp

```diff
@@ -93,7 +93,7 @@ sidb_simulation_result<Lyt> quicksim(const Lyt& lyt, const quicksim_params& ps =
         charge_lyt.assign_base_number(2);
         charge_lyt.assign_all_charge_states(sidb_charge_state::NEGATIVE);
         charge_lyt.update_after_charge_change(dependent_cell_mode::VARIABLE);
-        const auto negative_sidb_indices = charge_lyt.negative_sidb_detection();
+        const auto predefined_negative_sidbs_indices = charge_lyt.negative_sidb_detection();
```

Comment:
```suggestion
        const auto predefined_negative_sidb_indices = charge_lyt.negative_sidb_detection();
```

Reply:
you are right! That's indeed better. Thanks!